### ISSUE TITLE
Setting or getting multiple options at the same time crashes River

### ISSUE THREAD

**Issue by Bonicgamer** [ifreund/river#228]
I just found out that River has the option to create variables/options, but it seems it's a little  buggy.
One bug I found is that when setting or getting two or more options at the same time River crashes.
Example:
```
#!/bin/sh
# This crashes River
riverctl get-option tst &
riverctl get-option tst &
```

**Comment by Leon-Plickat** [comment]
I can reproduce.

But I don't think it is related to river-options. I accidentally tested with an older riverctl, which does not yet have river-options and just sends the `get-option tst` command to river and still get the crash. So I tried with a different command (`background-colo`) and can also reliably crash river.

**Comment by ifreund** [comment]
The bug here is in Control.zig. Using the id as the key for the hashmap is not sufficient as the same id may be used for different clients. Instead we should probably hash client and id together

**Comment by ifreund** [comment]
Fixed in 4beb39920a591d65287ad82de65483747f68f0c2
